File: Test6/simpleCipher.py
alphabet = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'

# def simpleCipher(encrypted, k):
#     return ''.join([alphabet[(alphabet.upper().index(c) - k)] for c in encrypted])

# def simpleCipher(encrypted, k):
#     encrypted = encrypted.upper()
#     decrypted = ''
#     for c in encrypted:
#         decrypted += alphabet[(alphabet.index(c) - k)]
#     return decrypted

def simpleCipher(encrypted, k):
    encrypted = encrypted.upper()
    decrypted = ''
    for c in encrypted:
        decrypted += alphabet[(alphabet.find(c) - k)]
    return decrypted
#This solutions is not expected big number of k

print(simpleCipher('VTAOG', 2))

File: Test6/simpleCipher.js
// const simpleCipher(encrypted, k) {

// }

let alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
let solve = "VTAOG";

let simpleCipher = (encrypted, k) => {
  encrypted = encrypted.trim().toUpperCase();
  let decrypted = "";
  for (let i = 0; i < encrypted.length; i++) {
    let index = alphabet.indexOf(encrypted[i]);
    let newIndex = index - k;
    if (newIndex < 0) newIndex += 26;
    decrypted += alphabet[newIndex];
  }
  return decrypted;
};

console.log(simpleCipher(solve, 2));
